portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionUsingConfigFile.testcase

```diff
@@ -88,9 +88,24 @@ definition {
 	@description = "Verify that the Workspace label is present when a client extension is created through osgi"
 	@priority = 3
 	test OsgiClientExtensionWorkspaceLabelIsPresent {
+		task ("Given config file for client extension of type custom element created") {
+			ClientExtension.createGlobalVariableClientExtensionType(type = "customElement");
+
+			ClientExtension.checkConfigFileCreatedProperly();
+		}
+
+		task ("When Go to applications/custom apps/client extensions") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Custom Apps",
+				panel = "Applications",
+				portlet = "Client Extensions");
+		}
 
-		// TODO LPS-171621 OsgiClientExtensionWorkspaceLabelIsPresent pending implementation
+		task ("Then (date column) and label (workspace) are present") {
```

Comment:
Copy the Given When Then from the ticket: https://issues.liferay.com/browse/LPS-174964